packages/eslint-plugin/tests/rules/no-unused-vars/no-unused-vars.test.ts

```diff
@@ -921,6 +921,17 @@ export declare namespace Foo {
     }
   }
 }
+    `,
+    `
+class Foo<T> {
```

Comment:
this syntax is not supported by prettier yet, can you use `noFormat`?

Reply:
@armano2 Upgrading Prettier to 2.7 should work.